### PR TITLE
update debugging.rst for python 3.7

### PR DESCRIPTION
**Pull request by natashawatkins**
This was kind of hard to test -- I've forgotten the option in `conf.py` that I need to change to make sure it doesn't use markdown for rendering

### CODE REVIEW

File: rst_files/debugging.rst [hunk 1]
```diff
@@ -194,18 +192,13 @@ Now there won't be any exception, but the plot won't look right
...
 .. code-block:: python3
     :class: no-execute
 
-    import numpy as np
-    import matplotlib.pyplot as plt
-    from IPython.core.debugger import Pdb
-
     def plot_log():
-        Pdb().set_trace()
+        breakpoint()
```
Comment (mmcky) [inline rst_files/debugging.rst]: hey @natashawatkins I presume the breakpoint is still interactive with `n` for next in the PDB debugger right?

**Comment by mmcky** [comment]
Do you mean `:class: no-execute` then the output code-block can be highlighted with `ipython`

File: rst_files/debugging.rst (hunk 1 above)
Comment (natashawatkins) [reply rst_files/debugging.rst]: Yes it is

**Comment by natashawatkins** [comment]
we can do that

File: rst_files/debugging.rst [hunk 2]
```diff
@@ -35,6 +35,14 @@ Debugging tools for Python vary across platforms, IDEs and editors
 
 Here we'll focus on Jupyter and leave you to explore other settings
 
+We'll need the following imports
+
+.. code-block:: python3
```
Comment (mmcky) [inline rst_files/debugging.rst]: this should be tagged as `ipython` with the magic

File: rst_files/debugging.rst (hunk 1 above)
Comment (mmcky) [reply rst_files/debugging.rst]: cool

**Comment by natashawatkins** [comment]
@mmcky should be updated now

**Comment by mmcky** [comment]
thanks @natashawatkins

**Pull request merged**